schema/crdb/dbinit.sql

```diff
@@ -639,7 +639,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.region (
 
     read_only BOOL NOT NULL,
 
-    deleting BOOL NOT NULL
+    deleting BOOL NOT NULL,
+
+    reservation_factor FLOAT NOT NULL
```

Comment:
This definitely could benefit from docs, and it would probably be worth updating the "size_used" field in this file too.

Reply:
13f2a0797 adds some notes here, lmk what you think.